### PR TITLE
[DOC-8595] Docs for ARM General Availability

### PR DESCRIPTION
**Pull request by mdlinville**
[DOC-8595] Docs for ARM General Availability in v23.1.14 and v23.2.0

Releases pages will not update until v23.1.14 and v23.2.0 are released

### Previews
[src/current/v22.2/install-cockroachdb-linux.md](https://deploy-preview-18196--cockroachdb-docs.netlify.app/docs/v22.2/install-cockroachdb-linux.html)
[src/current/v23.1/install-cockroachdb-linux.md](https://deploy-preview-18196--cockroachdb-docs.netlify.app/docs/v23.1/install-cockroachdb-linux.html)
[src/current/v23.2/install-cockroachdb-linux.md](https://deploy-preview-18196--cockroachdb-docs.netlify.app/docs/v23.2/install-cockroachdb-linux.html)

### CODE REVIEW

File: src/current/v23.2/install-cockroachdb-linux.md [hunk 1]
```diff
@@ -167,8 +167,9 @@ true
 
 CockroachDB runtimes built for the ARM architecture have the following limitations:
 
-- CockroachDB on ARM is in [Limited Access](https://www.cockroachlabs.com/docs/stable/cockroachdb-feature-availability#feature-availability-phases) in CockroachDB v22.2.13, and is **experimental** in all other versions. Experimental binaries and Docker images are not qualified for production use and not eligible for support or uptime SLA commitments.
 - Clusters with a mix of Intel and ARM nodes are untested. Cockroach Labs recommends that all cluster nodes have identical CockroachDB versions, hardware, and software.
+- Because of the recommendation to avoid a mix of Intel and ARM nodes, Cockroach Labs recommends against migrating an Intel cluster to ARM in place. Instead, consider using CockroachDB's migration tooling, such as [Changefeed sinks]({% link v23.2/changefeed-sinks.md %}), to move the workload to a new ARM cluster.
```
Comment (srosenberg) [inline src/current/v23.2/install-cockroachdb-linux.md:171]: This is probably a question for @mwang1026, but methinks changefeed as a migration tool is a bit too narrow to be given as general guidance.

Comment (mdlinville) [reply src/current/v23.2/install-cockroachdb-linux.md]: Done, PTAL

File: src/current/v22.2/install-cockroachdb-linux.md [hunk 2]
```diff
@@ -164,8 +164,9 @@ true
...
+- CockroachDB on ARM is **experimental** in CockroachDB v22.2 versions prior to v22.2.13, is in [Limited Access](https://www.cockroachlabs.com/docs/stable/cockroachdb-feature-availability#feature-availability-phases) in CockroachDB v22.2.13, and is Generally Available (GA) in v22.2.14 and above. Experimental binaries and Docker images are not qualified for production use and not eligible for support or uptime SLA commitments.
 - Clusters with a mix of Intel and ARM nodes are untested. Cockroach Labs recommends that all cluster nodes have identical CockroachDB versions, hardware, and software.
+- Because of the recommendation to avoid a mix of Intel and ARM nodes, Cockroach Labs recommends against migrating an Intel cluster to ARM in place. Instead, consider using CockroachDB's migration tooling, such as [Changefeed sinks]({% link v23.2/changefeed-sinks.md %}), to move the workload to a new ARM cluster.
```
Comment (srosenberg) [inline src/current/v22.2/install-cockroachdb-linux.md:169]: "...against migrating an Intel cluster to ARM in place" and vice versa?

Comment (srosenberg) [inline src/current/v22.2/install-cockroachdb-linux.md:169]: Should the reference to changefeed sinks be removed in light of backup/restore as is the case for 23.1.14 and 23.2.x?

**Review by srosenberg — APPROVED** [review]

File: src/current/v22.2/install-cockroachdb-linux.md (hunk 2 above)
Comment (mdlinville) [reply src/current/v22.2/install-cockroachdb-linux.md:169]: Whoops, I missed one. Thanks.

Comment (mdlinville) [reply src/current/v22.2/install-cockroachdb-linux.md]: Done.

**Comment by mdlinville** [comment]
@mwang1026 approved in Slack.

File: src/current/v23.2/install-cockroachdb-linux.md [hunk 3]
```diff
@@ -167,8 +167,8 @@ true
 
 CockroachDB runtimes built for the ARM architecture have the following limitations:
 
-- CockroachDB on ARM is in [Limited Access](https://www.cockroachlabs.com/docs/stable/cockroachdb-feature-availability#feature-availability-phases) in CockroachDB v22.2.13, and is **experimental** in all other versions. Experimental binaries and Docker images are not qualified for production use and not eligible for support or uptime SLA commitments.
```
Comment (shannonbradshaw) [inline src/current/v23.2/install-cockroachdb-linux.md:170]: A similar statement appears near the top of this page. Does that need to change as well?

File: src/current/v23.1/install-cockroachdb-linux.md [hunk 4]
```diff
@@ -167,8 +167,9 @@ true
 
 CockroachDB runtimes built for the ARM architecture have the following limitations:
 
-- CockroachDB on ARM is in [Limited Access](https://www.cockroachlabs.com/docs/stable/cockroachdb-feature-availability#feature-availability-phases) in CockroachDB v22.2.13, and is **experimental** in all other versions. Experimental binaries and Docker images are not qualified for production use and not eligible for support or uptime SLA commitments.
```
Comment (shannonbradshaw) [inline src/current/v23.1/install-cockroachdb-linux.md:170]: A similar statement appears near the top of this page. Does that need to change as well?

File: src/current/v22.2/install-cockroachdb-linux.md [hunk 5]
```diff
@@ -164,8 +164,9 @@ true
 
 CockroachDB runtimes built for the ARM architecture have the following limitations:
 
-- CockroachDB on ARM is in [Limited Access](https://www.cockroachlabs.com/docs/stable/cockroachdb-feature-availability#feature-availability-phases) in CockroachDB v22.2.13, and is **experimental** in all other versions. Experimental binaries and Docker images are not qualified for production use and not eligible for support or uptime SLA commitments.
```
Comment (shannonbradshaw) [inline src/current/v22.2/install-cockroachdb-linux.md:167]: A similar statement appears near the top of this page. Does that need to change as well?

**Review by shannonbradshaw — CHANGES_REQUESTED** [review]
Same question on each Linux install page. Otherwise, LGTM.

File: src/current/v23.1/install-cockroachdb-linux.md (hunk 4 above)
Comment (mdlinville) [reply src/current/v23.1/install-cockroachdb-linux.md:170]: Good catch! Yes.

File: src/current/v22.2/install-cockroachdb-linux.md (hunk 5 above)
Comment (mdlinville) [reply src/current/v22.2/install-cockroachdb-linux.md:167]: Good catch! Yes. I also will make a tweak to the 23.2 version of the file to align.

**Review by shannonbradshaw — APPROVED** [review]
LGTM

**Pull request merged**